README.md

```diff
@@ -46,4 +46,5 @@ See [example manifest (dynamic)](./example-dynamic.yaml) for more details.
   * `csi.storage.k8s.io/node-stage-secret-name` - a name of a secret containing `username` and `password` used to authenticate in *onlineconf-admin*. Can contain template variables `${pvc.name}`, `${pvc.namespace}`, `${pv.name}` and `${pvc.annotations['<ANNOTATION_KEY>']}`, see [Kubernetes CSI docs](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html#node-stage-secret) for more information. Recommended value is `${pvc.name}`.
   * `csi.storage.k8s.io/node-stage-secret-namespace` - a namespace of this secret. Can contain template variables `${pvc.namespace}` and `${pv.name}`. Recommended value is `${pvc.namespace}`.
   * `uri` - URI of *onlineconf-admin* instance
+  * `updateInterval` - polling interval for requests to *onlineconf-admin* instance (default: "10s")
```

Comment:
What about static provisioning? Should the documentation be fixed or `updateInterval` does not work with PV?

Reply:
It definitely does work with PV, moreover I tested this patch only with PV. It looks like nothing prevents it from being unable to work with dynamic provisioning, so I leave this line here also.